### ISSUE TITLE
Consistent fact CLI (Unable to get file fact via cli)

### ISSUE THREAD

**Issue by taranlu-houzz** [Fizzadar/pyinfra#527]
**Describe the bug**
I seem to be unable to get some facts via the cli. Specifically, I have been unable to use `file`, `directory`, and `link`. This might be operator error, but I am finding the cli usage to be very confusing (I think more specific examples in the documentation would help).

**To Reproduce**
I have tried the following permutations:
```
❯ pyinfra ---- fact file --path "/Users/----/.bashrc"
Usage: pyinfra [OPTIONS] INVENTORY OPERATIONS...
Try 'pyinfra --help' for help.

Error: no such option: --path
```

```
❯ pyinfra ---- fact file path="/Users/----/.bashrc"
--> Loading config...
--> pyinfra error: No fact: path=/Users/----/.bashrc
```

```
❯ pyinfra ---- fact file "/Users/----/.bashrc"
--> Loading config...
--> pyinfra error: No fact: /Users/----/.bashrc
```

```
❯ pyinfra ---- fact.file "/Users/----/.bashrc"
--> Loading config...
--> pyinfra error: No such module: fact
```

```
❯ pyinfra ---- fact file
--> Loading config...
--> Loading inventory...

--> Connecting to hosts...
    [----] Connected

--> Gathering facts...
--> An unexpected internal exception occurred:

  File "/Users/----/.virtualenvs/05-hz-pyinfra-d0Faa_8y-py3.9/lib/python3.9/site-packages/pyinfra/api/facts.py", line 123, in _make_command
    return command_attribute(**host_args)
TypeError: command() missing 1 required positional argument: 'path'

--> The full traceback has been written to pyinfra-debug.log
--> If this is unexpected please consider submitting a bug report on GitHub, for more information run `pyinfra --support`.
```

The output from the final one `pyinfra <host> fact file` seems to imply that this should be possible, and the way that other facts work, such as `pyinfra <host> fact hostname` also seems to indicate this should work.

**Expected behavior**
I would expect all facts to be valid both inside of deploys as well as on the cli.

As a side note, I feel like it is a bit confusing that the way to call facts vs operations is different on the cli (e.g. `pyinfra <host> fact <fact name>` vs `pyinfra <host> <operation category>.<operation> <args, kwargs, etc>`).

**Comment by Fizzadar** [comment]
In this specific case the correct (and confusing) CLI would be `pyinfra --- fact file:/path/to/fle`. I agree that this is totally inconsistent with direct operation calls and in need of change! https://github.com/Fizzadar/pyinfra/issues/491 defines the change of facts to use explicit imports, based on that a better CLI might be:

```sh
pyinfra --- fact files.File path=/path/to/file
```

However - there is a limitation here in that each CLI call can only retrieve one fact - the same limitation applies to executing operations. This is also still not 100% consistent with the operation version (`fact files.File ...` vs `files.file ...`).

**Comment by Fizzadar** [comment]
This has now been released in `v1.4`!
